test/manipulation.test.js

```diff
@@ -1870,7 +1870,7 @@ describe('manipulation', function() {
     });
 
     bdd.itIf(connectorCapabilities.supportStrictDelete !== false &&
-      connectorCapabilities.cloudantCompatible,
+      connectorCapabilities.cloudantCompatible !== false,
```

Comment:
`&&` is used because if either `connectorCapabilities.supportStrictDelete` OR `connectorCapabilities.cloudantCompatible` is set to `false` the test case should be unsupported. Using `||` would mean both would have to be `false` for the test to be unsupported. They're unrelated compatibility flags.